develop/api_guc.rst

```diff
@@ -277,6 +277,21 @@ Specifies whether to automatically propagate DDL changes from the coordinator to
 
   For a list of DDL propagation support, see :ref:`ddl_prop_support`.
 
+.. _enable_local_ref_fkeys:
+
+citus.enable_local_reference_table_foreign_keys (boolean)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+Enables foreign keys between reference tables and local tables when coordinator is added to metadata (see :ref:`citus_add_node`).
```

Comment:
Oh also, are you sure we need to citus_add_node the coordinator? Does single-node Citus remove that requirement?

Reply:
>Citus must convert the plain table into a "citus-local" table

In 9.5, we were calling naming such tables as "citus-local tables" but in 10.0, we decided to call them as "local tables added to metadata" since we actually don't want to draw a picture as if we introduced another table type. So, I think we should not use "citus-local" thing anywhere.

>Also, what are the limitations exactly?

The answer to this question would be a little bit long. This is because, for example any DDL limitation applying to reference tables (except creating/dropping triggers) do also apply to those local tables that are added to metadata.
But I can tell that we use more or less the same components to process queries & utility commands on those local tables with reference tables.

> Does single-node Citus remove that requirement?

In Citus 10.0, we implicitly add coordinator only if `pg_dist_node` is empty. So I think we should tell that user should explicitly add coordinator to metadata to be able to define such foreign keys.

Reply:
OK, want to add my suggested text but with the new terminology for citus-local?

Reply:
sure